target/linux/ath79/dts/qca9558_mikrotik_routerboard-96x.dtsi

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca955x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mikrotik,routerboard-962uigs-5hact2hnt", "qca,qca9558";
+	model = "MikroTik RouterBOARD 962UiGS-5HacT2HnT (hAP ac)";
```

Comment:
Remove from `.dtsi`

Reply:
@rmounce thanks for replying. Could you rebase your change so the OpenWRT community will be able to merge?

Reply:
Rebased and re-tested

Reply:
@rmounce thanks.
@f00b4r0  could you take a look on that PR please?